Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN git clone ${PETSC_URL} /petsc
 WORKDIR /petsc
 RUN git checkout ${PETSC_VERSION}
 
+# Apply the fix from Jed's branch
+RUN git checkout 8d1aee66 config/BuildSystem/config/packages/tchem.py
```

Comment:
Dr. Knepley was able to get the TChem configure fix in the main branch and should be in release soon.  Depending upon when this makes it into release we may want to wait to merge in this PR.